.github/actions/spell-check/expect/a129ff14ec985d6b7bf09e296a831c955d41040b.txt

```diff
@@ -0,0 +1,14 @@
+checkboxes
```

Comment:
was this a file that git added accidentally?

Reply:
This text file is added with the scripted provided by the bot reply above. 
it helps the github recognize new names to reduce the spell checking warnings.